ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/InfoMapper.java

```diff
@@ -202,31 +211,66 @@ private static String normalizeTitle(String title) {
     private static OASResult parseServiceInfoAnnotationAttachmentDetails(List<OpenAPIMapperDiagnostic> diagnostics,
                                                                          AnnotationNode annotation,
                                                                          Path ballerinaFilePath) {
+
         Location location = annotation.location();
         OpenAPI openAPI = new OpenAPI();
-        Optional<MappingConstructorExpressionNode> content = annotation.annotValue();
-        // If contract path there
-        if (content.isPresent()) {
-           SeparatedNodeList<MappingFieldNode> fields = content.get().fields();
-           if (!fields.isEmpty()) {
-               OpenAPIInfo openAPIInfo = updateOpenAPIInfoModel(fields);
-               // If in case ballerina file path is getting null, then openAPI specification will be generated for
-               // given services.
-               if (openAPIInfo.getContractPath().isPresent() && ballerinaFilePath != null) {
-                   return updateExistingContractOpenAPI(diagnostics, location, openAPIInfo, ballerinaFilePath);
-               } else if (openAPIInfo.getTitle().isPresent() && openAPIInfo.getVersion().isPresent()) {
-                   openAPI.setInfo(new Info().version(openAPIInfo.getVersion().get()).title(normalizeTitle
-                           (openAPIInfo.getTitle().get())));
-               } else if (openAPIInfo.getVersion().isPresent()) {
-                   openAPI.setInfo(new Info().version(openAPIInfo.getVersion().get()));
-               } else if (openAPIInfo.getTitle().isPresent()) {
-                   openAPI.setInfo(new Info().title(normalizeTitle(openAPIInfo.getTitle().get())));
-               }
-           }
+        Optional<MappingConstructorExpressionNode> contentOpt = annotation.annotValue();
```

Comment:
```suggestion
        Optional<MappingConstructorExpressionNode> svcInfoAnnotationValue = annotation.annotValue();
```

Wouldn't this be more meaningful in this context ?